terraform/script.py

```diff
@@ -63,28 +59,76 @@ def get_envs():
     nodes_number = os.getenv("NODES_NUMBER")
     print(cluster_name)
 
-def confirm_config():
-    True
+
+def select_workspace():
+    global kube
+    if os.getenv("TF_VAR_CLOUDPROVIDER") == "aws":
+        kube = str("eks")
+    elif os.getenv("TF_VAR_CLOUDPROVIDER") == "gcp":
+        kube = str("gke")
+    elif os.getenv("TF_VAR_CLOUDPROVIDER") == "az":
+        kube = str("azkube")
+    
+    os.environ["TF_VAR_KUBE"] = kube
+
+        
+    output = str(subprocess.check_output("cd live/$TF_VAR_MQ/$TF_VAR_CLOUDPROVIDER/"+kube+" ; terragrunt workspace list", shell=True))
```

Comment:
I think using `f-string` is cleaner way of string formatting. I will look something like this `f"cd live/$TF_VAR_MQ/$TF_VAR_CLOUDPROVIDER/{kube}"` You can later use it as well in defining bash_commands

Reply:
Corrected.